sba-business-rules.md

```diff
@@ -0,0 +1,37 @@
+---
+title: DATA Act business rules
+---
+As an SBA user, I need to see the business rules being applied to my source data
+
+Business rules or validation rules will be applied to ensure that submitted data by the US Small Business Administration meets the DATA Act criteria. Goals of these validation rules will improve cleanliness, accuracy, and meaningfulness of the data set. Below is a draft of proposed rules and are subject to change. 
+
+*Standard validation*
+
+Required fields: 
+	Required field validation ensures that users input data in required fields. If field is empty, an error will be raised. Required fields to be determined in the [DATA act schema](https://github.com/18F/intercessor/blob/master/schema/data-act-schema.png)
+
+Data types: 
+	Data type validation ensures that the data types match with the [DATA Act schema](https://github.com/18F/intercessor/blob/master/schema/data-act-schema.png) . For example, `parentAward` data type required is a string, If a number is entered an error will be raised. 
+
+
+*Awardee validation:*  
```

Comment:
Good idea, having a separate section for awardee validation. What do you think about breaking out the individual validations as a list so they're easier to read?